lambda-search/deploy_lambda.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+source keys.sh
+
+cargo lambda build --release --bin lambda --arm64 --output-format zip
+
+case $EMBEDDING in
+("COHERE"|"OPEN_AI"))
+  export EMBEDDING_ENV="EMBED_API_KEY=$EMBED_API_KEY"
+  ;;
+"MIGHTY")
+  export EMBEDDING_ENV="MIGHTY_URI=$MIGHTY_URI"
+  ;;
+esac
+
+# Clear previous versions
+aws lambda delete-function-url-config --region $LAMBDA_REGION --function-name $LAMBDA_FUNCTION_NAME
+aws lambda delete-function --region $LAMBDA_REGION --function-name $LAMBDA_FUNCTION_NAME
+
+# Deploy to AWS Lambda
+aws lambda create-function --function-name $LAMBDA_FUNCTION_NAME \
+  --handler bootstrap \
+  --architectures arm64 \
+  --zip-file fileb://./target/lambda/page-search/bootstrap.zip \
```

Comment:
Why do we expect the subdirectory to be `page-search`, not `lambda`?

Reply:
Because the project used to be `page-search` and the `lambda.rs` was formerly `main.rs` (in which case `cargo lambda` picks up the project name). Fixed.